src/support/z_context.erl

```diff
@@ -187,6 +187,7 @@ new(Req, Module) when is_map(Req) ->
     Context#context{controller_module=Module}.
 
 
+-spec set_default_language_tz(#context{}) -> #context{}.
 set_default_language_tz(Context) ->
     F = fun() -> {z_language:default_language(Context), tz_config(Context)} end,
     {DefaultLang, TzConfig} = z_depcache:memo(F, default_language_tz, ?DAY, [config], Context),
```

Comment:
Line 193 is too long:     {DefaultLang, TzConfig} = z_depcache:memo(F, default_language_tz, ?DAY, [config], Context),.